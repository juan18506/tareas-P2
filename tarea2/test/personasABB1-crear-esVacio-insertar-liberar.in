crearArbolPersonas

esVacioArobolPersonas

liberarArbolPersonas

crearArbolPersonas

crearAgenda
crearPersona 3 33 Persona3
insertarPersonaArbolPersonas

esVacioArobolPersonas

crearAgenda
crearPersona 1 11 Persona1
insertarPersonaArbolPersonas

esVacioArobolPersonas

crearAgenda
crearPersona 2 22 Persona2
crearFecha 1/12/22
crearEvento 1 Descripcion1
agregarEventoPersona
insertarPersonaArbolPersonas

esVacioArobolPersonas

liberarArbolPersonas

Fin