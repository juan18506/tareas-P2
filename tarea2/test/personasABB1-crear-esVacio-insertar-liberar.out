1>2>El árbol de personas está vacío.
3>4>5>6>7>8>El árbol de personas NO está vacío.
9>10>11>12>El árbol de personas NO está vacío.
13>14>15>16>17>18>19>El árbol de personas NO está vacío.
20>21>Fin.
